src/kartograf/atom_mapper.py

```diff
@@ -101,12 +103,15 @@ def __init__(
         _mapping_algorithm : str, optional
             mapping_algorithm.linear_sum_assignment - this allows to swap the
             optimization algorithm. Not recommended.
+        allow_partial_fused_rings: bool
```

Comment:
I think it might be good to be careful about the proliferation of kwarg flags over just passing things to "additional_mapping_filter_functions".

I assume the intent here is to have this be explicit because it's core to how our RFE Protocols work, but it might be good to keep in mind what we want this signature to look like in the future.

Reply:
Good point I can see this getting out of hand over time for many optional things, but as this is a recommended best practice we want this to be easy to turn on.